WooCommerce/Classes/ViewRelated/Hub Menu/HubMenu.swift

```diff
@@ -27,7 +27,7 @@ struct HubMenu: View {
 
             // Store Section
             Section {
-                Button {
+                ListPressableButton {
```

Comment:
Went for the solution where we just change Button to ListPressableButton and get the proper color :)